docs/oscar_references.bib

```diff
@@ -501,3 +501,34 @@ @book{Marcus
        URL = {https://doi.org/10.1007/978-3-319-90233-3},
 }
 
+@incollection{GJ00,
+   author =       {Ewgenij Gawrilow and Michael Joswig},
+   title =        {polymake: a Framework for Analyzing Convex Polytopes},
+   pages =        {43-74},
+   editor =       {Gil Kalai and G\"{u}nter M. Ziegler},
+   booktitle =    {Polytopes --- Combinatorics and Computation},
+   publisher =    {Birkh\"{a}user},
+   year =         {2000},
+    URL = {https://doi.org/10.1007/978-3-0348-8438-9_2},
+    NOTE = {https://polymake.org}
+}
+
+@InProceedings{KLT20,
+   author="Kaluba, Marek
+      and Lorenz, Benjamin
+      and Timme, Sascha",
+   editor="Bigatti, Anna Maria
+      and Carette, Jacques
+      and Davenport, James H.
+      and Joswig, Michael
+      and de Wolff, Timo",
+   title="Polymake.jl: A New Interface to polymake",
+   booktitle="Mathematical Software -- ICMS 2020",
+   year="2020",
+   publisher="Springer International Publishing",
+   address="Cham",
+   pages="377--385",
+   abstract="We present the Julia interfaceto polymake, a software for research in polyhedral geometry. We describe the technical design and how the integration into Julia makes it possible to combine polymake with state-of-the-art numerical software.",
```

Comment:
```suggestion
   abstract="We present the Julia interface to polymake, a software for research in polyhedral geometry. We describe the technical design and how the integration into Julia makes it possible to combine polymake with state-of-the-art numerical software.",
```

Reply:
Thanks for catching this. This is actually the original bibtex from the Springer page, amazing.